app/javascript/views/walletServices/StripeConnection.tsx

```diff
@@ -0,0 +1,134 @@
+import axios from "axios";
+import * as React from "react";
+import { FormattedMessage, injectIntl } from "react-intl";
+
+import routes from "../routes";
+
+import { LoaderIcon } from "brave-ui/components/icons";
+import { FlexWrapper } from "../style";
+
+import Modal, { ModalSize } from "../../components/modal/Modal";
+
+interface IStripeConnectionState {
+  detailsSubmitted: false;
+  displayName: string;
+  isLoading: boolean;
+  payoutsEnabled: boolean;
+  stripeUserId: string;
+}
+
+class StripeConnection extends React.Component<any, IStripeConnectionState> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      detailsSubmitted: false,
+      displayName: "",
+      isLoading: false,
+      payoutsEnabled: false,
+      stripeUserId: null,
+    };
+  }
+
+  public componentDidMount() {
+    this.reload();
```

Comment:
Could this data get passed in as a prop from <WalletServices/> ? Although, Maybe it makes more sense to do it in this file, as not every user may have a Stripe connection

Reply:
In the designs these are somewhat separated. I think I could likely abstract out the data layer 🤔 the `/wallet` endpoint which is called in `BraveConnection` is loading quite a bit of data. I think I want to keep this for right now.